### DISCUSSION TITLE
Move this repo to TrustTheVote-Project sooner rather than later

### DISCUSSION THREAD

**Discussion by stratofax** [ion-oset/developer-commons#12] [Meta]
As I add new discussions and issues to this repo, I've included some links to other posts in this repo. These links include the full path to the item, including the repo name. I'm concerned that these links will break when this repo is transferred to [The TrustTheVote® Project](https://github.com/TrustTheVote-Project).

To minimize potential linkrot, I think we should transfer this repo to [The TrustTheVote® Project](https://github.com/TrustTheVote-Project) as soon as possible.

Example: [OSET Licence, formatted with Markdown · Discussion #10 · ion-oset/developer-commons](https://github.com/ion-oset/developer-commons/discussions/10) contains:

- an external link to another repo, which presumably will migrate without a problem,
- an internal link which may break when this repo transfers [The TrustTheVote® Project](https://github.com/TrustTheVote-Project),
- a file link, which may also break. 

Perhaps GitHub is smart enough to update all the links to this repo when it's transferred to another owner, but that seems unlikely.

Also, the sooner this repo moves to [The TrustTheVote® Project](https://github.com/TrustTheVote-Project), the more we can solicit contributions from other OSET developers.

**Comment by ion-oset** [comment]
The logistics of a moving a repository to a new owner are nominally simple: [Transferring a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository)

> Prerequisites for repository transfers:
> 
> - [ ] When you transfer a repository that you own to another user account, the new owner will receive a confirmation email. The confirmation email includes instructions for accepting the transfer. If the new owner doesn't accept the transfer within one day, the invitation will expire.
> - **N/A** ~To transfer a repository that you own to an organization, you must have permission to create a repository in the target organization.~
> - [x] The target account must not have a repository with the same name, or a fork in the same network.
> - [x] The original owner of the repository is added as a collaborator on the transferred repository. Other collaborators to the transferred repository remain intact.
> - **N/A** ~Private forks can't be transferred.~


The checked items are going to handle themselves. The unchecked item needs [The TrustTheVote® Project](https://github.com/TrustTheVote-Project) to accept the transfer.

I agree that the links will probably break. But if they do since wikis can be cloned locally it should be possible to do a cleanup of the links using something like:

```
sed 's|https://github.com/OLD-OWNER|https://github.com/NEW-OWNER|g' WikiPage.md
git commit -m "Rename wiki links to match new repository."
git push
```

**Reply by stratofax** [reply]
The wiki and the source files will be easy to clean up using the commands you provided; I was thinking more in terms of the Discussions and Issues -- I don't think they allow global find-and-replace operations (but I may be wrong).

The important thing, IMHO, is to err on the side of transferring sooner rather than later, without creating a false sense of urgency, since we have a number of actual deadlines on the radar right now. Partly, this is about the links, but also because I think the developer commons is valuable and I want to share the benefits with the rest of the OSET team, too!
